sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/spanner/SpannerIO.java

```diff
@@ -784,11 +785,13 @@ public PDone expand(PCollection<MutationGroup> input) {
       if (sampler == null) {
         sampler = createDefaultSampler();
       }
+      PCollection<Void> create = input
```

Comment:
Subjectively I think this would look better if completely inlined into the statement at line 793.

Also could you add a comment explaining why we need to wait on the input?